src/organizerproxy.cpp

```diff
@@ -22,10 +22,31 @@ OrganizerProxy::OrganizerProxy(OrganizerCore* organizer, PluginContainer* plugin
   : m_Proxied(organizer)
   , m_PluginContainer(pluginContainer)
   , m_Plugin(plugin)
+  , m_PluginName(plugin->name())
   , m_DownloadManagerProxy(std::make_unique<DownloadManagerProxy>(this, organizer->downloadManager()))
   , m_ModListProxy(std::make_unique<ModListProxy>(this, organizer->modList()))
   , m_PluginListProxy(std::make_unique<PluginListProxy>(this, organizer->pluginList()))
 {
+  m_Connections.push_back(m_Proxied->onAboutToRun(callSignalIfPluginActive(this, m_AboutToRun, true)));
+  m_Connections.push_back(m_Proxied->onFinishedRun(callSignalIfPluginActive(this, m_FinishedRun)));
+  m_Connections.push_back(m_Proxied->onProfileCreated(callSignalIfPluginActive(this, m_ProfileCreated)));
+  m_Connections.push_back(m_Proxied->onProfileRenamed(callSignalIfPluginActive(this, m_ProfileRenamed)));
+  m_Connections.push_back(m_Proxied->onProfileRemoved(callSignalIfPluginActive(this, m_ProfileRemoved)));
+  m_Connections.push_back(m_Proxied->onProfileChanged(callSignalIfPluginActive(this, m_ProfileChanged)));
+
+  m_Connections.push_back(m_Proxied->onUserInterfaceInitialized(callSignalAlways(m_UserInterfaceInitialized)));
+  m_Connections.push_back(m_Proxied->onPluginSettingChanged(callSignalAlways(m_PluginSettingChanged)));
+  m_Connections.push_back(m_Proxied->onPluginEnabled(callSignalAlways(m_PluginEnabled)));
+  m_Connections.push_back(m_Proxied->onPluginDisabled(callSignalAlways(m_PluginDisabled)));
+
+}
+
+OrganizerProxy::~OrganizerProxy()
+{
+  log::debug("~OrganizerProxy() for {}.", m_PluginName);

```

Comment:
Is this log necessary?

Reply:
I had some issues with `OrganizerProxy` not being released, especially with Python plugins (see the `unload` PR). The proxies are the ones that disconnect the slots, so they're not released bad stuff can happen and it's easier to debug. I can change the message to something like *"Disconnecting signals for plugin X."* which is more the intended purpose.

Reply:
Lots of people run MO with debug logs enabled at all times, just trying to cut down on unnecessary ones. You can leave it there if you need it, especially for devbuilds.

Reply:
I've changed the message. Note that this should rarely appear on standard MO2 usage (except at the end when closing) since it's most likely plugin developers that will reload / unload plugins.